sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -228,6 +232,7 @@ export class KeyClient {
     listPropertiesOfKeys(options?: ListPropertiesOfKeysOptions): PagedAsyncIterableIterator<KeyProperties>;
     listPropertiesOfKeyVersions(name: string, options?: ListPropertiesOfKeyVersionsOptions): PagedAsyncIterableIterator<KeyProperties>;
     purgeDeletedKey(name: string, options?: PurgeDeletedKeyOptions): Promise<void>;
+    releaseKey(name: string, version: string, target: string, options?: ReleaseKeyOptions): Promise<ReleaseKeyResult>;
```

Comment:
IIRC, the target is the environment they want to target. We might use `environment` instead, but talk with Hervey.

Reply:
Or maybe even `targetEnvironment`. Would be nice to align with the swagger, even if it means changing that (it's in preview).